remote-api/src/test/java/org/alfresco/rest/api/impl/CategoriesImplTest.java

```diff
@@ -628,6 +684,30 @@ public void testGetCategoryChildren_includeCount()
             .isEqualTo(List.of(0, 2, 0));
     }
 
+    @Test
+    public void testGetCategoryChildren_includePath()
+    {
+        final NodeRef parentCategoryNodeRef = createNodeRefWithId(PARENT_ID);
+        given(nodesMock.validateNode(PARENT_ID)).willReturn(parentCategoryNodeRef);
+        given(nodesMock.isSubClass(parentCategoryNodeRef, ContentModel.TYPE_CATEGORY, false)).willReturn(true);
+        final int childrenCount = 3;
+        final List<ChildAssociationRef> childAssociationRefMocks = prepareChildAssocMocks(childrenCount, parentCategoryNodeRef);
+        given(nodeServiceMock.getChildAssocs(parentCategoryNodeRef)).willReturn(childAssociationRefMocks);
+        childAssociationRefMocks.forEach(this::prepareCategoryNodeMocks);
+        given(parametersMock.getInclude()).willReturn(List.of(Nodes.PARAM_INCLUDE_PATH));
+        given(nodeServiceMock.getPath(any())).willReturn(mockCategoryPath());
+
+        // when
+        final List<Category> actualCategoryChildren = objectUnderTest.getCategoryChildren(PARENT_ID, parametersMock);
+
+        assertThat(actualCategoryChildren)
+                .isNotNull()
+                .hasSize(3)
```

Comment:
[Really minor] This line makes the test slightly less useful since if the sizes differ then we don't get to find out what the values are.

Reply:
Thanks, noted